File: docs/ide/codesnippet/CSharp/step-9-review-comment-and-test-your-code_1.cs
    private void showButton_Click(object sender, EventArgs e)
    {
        // Show the Open File dialog. If the user clicks OK, load the
        // picture that the user chose.
        if (openFileDialog1.ShowDialog() == DialogResult.OK)
        {
            pictureBox1.Load(openFileDialog1.FileName);
        }
    }

File: docs/ide/codesnippet/VisualBasic/step-9-review-comment-and-test-your-code_1.vb
    Private Sub showButton_Click() Handles showButton.Click

        ' Show the Open File dialog. If the user clicks OK, load the
        ' picture that the user chose.
        If OpenFileDialog1.ShowDialog() = DialogResult.OK Then
            PictureBox1.Load(OpenFileDialog1.FileName)
        End If

    End Sub